build/Targets/GenerateCompilerInternals.targets

```diff
@@ -35,7 +35,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-	<MonoPrefix Condition="'$(OS)' != 'Windows_NT'">$(MonoToolsetPath) </MonoPrefix>
+      <MonoPrefix Condition="'$(OS)' != 'Windows_NT'">mono</MonoPrefix>
```

Comment:
Yes, just to run the generators. Converting the generators over to netcore is my next task.